lib/app.rb

```diff
@@ -0,0 +1,152 @@
+require_relative 'classroom'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
```

Comment:
**Highlights**
- Great job using require_relative to include your files

Reply:
Thank you!